Tests/Marketplace/override_core_packs_versions.py

```diff
@@ -0,0 +1,127 @@
+import os
+import argparse
+import uuid
+import json
+
+from Tests.Marketplace.marketplace_services import json_write, get_content_git_client
+from Tests.Marketplace.marketplace_constants import GCPConfig, CONTENT_ROOT_PATH
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+
+
+def should_override_locked_corepacks_file(marketplace: str, last_upload_commit: str):
+    """
+        Checks if the corepacks_override.json file in the repo should be used to override an existing corepacks file.
+        The override file should be used if the following conditions are met:
+        1. The current version of the corepacks_override.json is different from the last version in master.
+        2. The current marketplace appears as should be overriden in the corepacks_override.json.
+
+        Args
+            marketplace (str): the marketplace type of the bucket. possible options: xsoar, xsoar_saas, marketplace_v2 or xpanse
+            last_upload_commit(str): Last upload commit.
+
+        Returns True if a file should be updated and False otherwise.
+        """
+    # Specify the file path
+    file_path = "Tests/Marketplace/corepacks_override.json"
+
+    content_repo = get_content_git_client(CONTENT_ROOT_PATH)
+    logging.info(last_upload_commit)
+    commit = content_repo.commit(last_upload_commit)
+
+    # Access the file as a blob from the last commit
+    last_commit_blob = commit.tree / file_path
+
+    # Get the content of the last commit blob
+    last_commit_content = json.loads(last_commit_blob.data_stream.read())
+
+    # Get the current file content
+    current_override_content = GCPConfig.corepacks_override_contents
+
+    # If the files are different and the current marketplace is in the override file we override
+    if current_override_content != last_commit_content and marketplace in current_override_content:
```

Comment:
shouldn't it be:
```suggestion
    if current_override_content != last_commit_content and marketplace in current_override_content.get("marketplaces", []):
```

Reply:
no, the new formation is as follows:
server_verions: x_x_x
xsoar_saas: {
      updated_corepacks_content: {...}
}